src/modules/px4iofirmware/mixer.cpp

```diff
@@ -167,9 +180,10 @@ mixer_tick(void)
 	 * FMU or from the mixer.
 	 *
 	 */
+	bool effective_safety_off = (r_status_flags & PX4IO_P_STATUS_FLAGS_SAFETY_OFF) != 0 || override_enabled;
```

Comment:
@tridge How does the in-air restart work for APM? Are the PWM kept at the last value until the FMU comes back, or are they switched off?
I'm trying to understand why this change is necessary.